package.json

```
@@ -1,6 +1,6 @@
 {
 	"name": "voicemeeter-connector",
-	"version": "1.4.1",
+	"version": "1.5.0",

```

maybe lets introduce a breaking change, since this change does a lot in the background.
```suggestion
	"version": "2.0.0",
```